### PR TITLE
xsettingsd: 1.0.0 -> 1.0.2

### PR DESCRIPTION
**Pull request by peterhoeg**
###### Motivation for this change

Also use `cmake` as it means we have to patch less and get more things done for us automatically.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 112609` at d6a82ff0 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>libsForQt512.kde-gtk-config</li>
    <li>libsForQt514.kde-gtk-config</li>
  </ul>
</details>
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/fe3a582c4325f40c0bbe82ca3a0a2c32">libsForQt5.kde-gtk-config</a></li> (libsForQt515.kde-gtk-config ,plasma5Packages.kde-gtk-config)</li>
    <li><a href="https://gist.github.com/e094120117d35595bd0c294e22be9f52">xsettingsd</a></li></li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 112609` at d6a82ff0 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>libsForQt512.kde-gtk-config</li>
    <li>libsForQt514.kde-gtk-config</li>
  </ul>
</details>
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/6e3c86bf2e15efbd977d3480dc4be808">libsForQt5.kde-gtk-config</a></li> (libsForQt515.kde-gtk-config ,plasma5Packages.kde-gtk-config)</li>
    <li><a href="https://gist.github.com/57e32c673efea24540c4086104487559">xsettingsd</a></li></li>
  </ul>
</details>

File: pkgs/tools/X11/xsettingsd/default.nix [hunk 1]
```diff
@@ -1,38 +1,31 @@
...
-  buildPhase = ''
-    scons -j$NIX_BUILD_CORES -l$NIX_BUILD_CORES
-  '';
-
-  installPhase = ''
-    install -D -t "$out"/bin xsettingsd dump_xsettings
-    install -D -t "$out"/usr/share/man/man1 xsettingsd.1 dump_xsettings.1
+  # we end up with symlinked unit files if we don't move them around ourselves
+  postFixup = ''
+    rm -rf $out/lib/systemd
```
Comment (SuperSandro2000) [inline pkgs/tools/X11/xsettingsd/default.nix:20]:
```suggestion
    rm -r $out/lib/systemd
```

**Review by romildo — APPROVED** [review]
Looks good to me. Tested locally.

**Pull request merged**